### DISCUSSION TITLE
Walk-through for NICE Challenge: Network Administration Crash Course [NG]

### DISCUSSION THREAD

**Discussion by obrazzy55** [community/community#108074] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can I get Walkthrough for NICE Challenge: Network Administration Crash Course [NG]
Author: Bailey Kasin & Robert Hancock
Framework Category: Operate and Maintain
Specialty Area: Network Services
Work Role: Network Operations Specialist
Task Description: Diagnose network connectivity problem. (T0081)
Scenario
Daswebs is in desperate need of a new employee to accomplish a range of tasks typically required of a network administrator. Your goal is to accomplish as many of the requests that are provided by your co-workers as possible.

Additional Information
More details and objectives about this challenge will be introduced during the challenge meeting, which will start once you begin deploying the challenge.

**Comment by queenofcorgis** [comment]
Hi @obrazzy55 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this challenge. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
